benchmark/fluid/fluid_benchmark.py

```diff
@@ -266,7 +266,10 @@ def train(avg_loss, infer_prog, optimizer, train_reader, test_reader, batch_acc,
             # FIXME(wuyi): For use_reader_op, if the current
             # pass is not the last, the last batch of this pass
             # is also equal to args.batch_size.
-            num_samples += len(args.batch_size)
+            if args.use_reader_op:
+                num_samples += args.batch_size
```

Comment:
`args.batch_size` is the batch size on each GPU now. So it should be ` num_samples += args.batch_size * args.gpus`

Reply:
Thanks very much! Done. Current know issue, if set `--use_reader_op` we must also set `--no_test` will fix this in next PR.